app/api/private-cloud/products/[licencePlate]/comments/_operations/update.ts

```diff
@@ -1,3 +1,37 @@
-export default async function updateOp() {
-  return { message: 'Update comment successful' };
+import prisma from '@/core/prisma';
+
+interface UpdateCommentParams {
+  licencePlate: string;
+  commentId: string;
+  text: string;
+}
+
+export async function updateOp({ licencePlate, commentId, text }: UpdateCommentParams) {
+  const project = await prisma.privateCloudProject.findUnique({
+    where: { licencePlate },
+  });
+
+  if (!project) {
+    throw new Error('Project not found');
+  }
+
+  const updatedComment = await prisma.privateCloudComment.updateMany({
+    where: {
+      AND: [{ id: commentId }, { projectId: project.id }],
```

Comment:
`AND` is not necessary and you might not need to fetch the project beforehand, takes a look at the query of `list` op.